pkg/GAPJulia/JuliaInterface/julia/gap_to_julia.jl

```diff
@@ -9,11 +9,20 @@ The parameter `recursion_dict` is meant to preseve egality
 of converted objects and should never be given by the user.
 """
 
+## Default for conversion:
+## Do not convert Julia types, as long as type matches
```

Comment:
It would be nice if the comment also explained why this is useful, e.g. by describing an example use case.

Reply:
I still find this confusing... We *do* convert them, just to themselves. Or, for yet another perspective: we do not convert them, but still let them through, while other objects are explicitly rejected.
```suggestion
## Base case fro conversion (least specialized method): Allow converting any
## Julia object x to type T, provided that the type of x is a subtype of T;
## otherwise, explicitly reject the conversion.
```